main/lsp/requests/definition.cc

```diff
@@ -9,6 +11,26 @@ DefinitionTask::DefinitionTask(const LSPConfiguration &config, MessageId id,
                                unique_ptr<TextDocumentPositionParams> params)
     : LSPRequestTask(config, move(id), LSPMethod::TextDocumentDefinition), params(move(params)) {}
 
+core::Loc
+DefinitionTask::findRequireRelativeLoc(const core::GlobalState &gs,
+                                       const std::vector<std::unique_ptr<core::lsp::QueryResponse>> &responses) {
+    if (responses.size() < 2)
+        return core::Loc::none();
```

Comment:
```suggestion
    if (responses.size() < 2) {
        return core::Loc::none();
    }
```

Reply:
@jez any chance to encode that in `format_cxx.sh`?

Reply:
Happy to if you can find the setting. I don’t think that clang-format has such an option and I don’t feel like fussing with clang-tidy in bazel (in three years of reviewing Sorbet PRs this is the first time I’ve seen someone use no braces, so ROI is probably low)

Reply:
A cursory glance at the Googling literature seems to indicate `clang-format` doesn't support this (or that you have to combine it with another tool/clang-tidy to add the braces in the first place). I'll keep it in mind in the future (maybe there is a VSCode side change I can do).